odoo/addons/base/models/ir_model.py

```diff
@@ -1682,7 +1682,7 @@ class IrModelAccess(models.Model):
     _description = 'Model Access'
     _order = 'model_id,group_id,name,id'
 
-    name = fields.Char(required=True, index=True)
+    name = fields.Char(compute='_compute_name', store=True, readonly=False)
```

Comment:
```suggestion
    name = fields.Char(compute='_compute_name', store=True, readonly=False, required=True, precompute=True)
```
Keep it required please, we do not want to potentially have records without names.

Reply:
> Keep it required please, we do not want to potentially have records without names.

But why? It is completely useless. It doesn't have any meaning.

Reply:
Actually `name` has never been used in the code, and anywhere we have ACL in the views it is a tree view (maybe because name is not reliable or meaningful, and it is not unique)